macros/common/stage_external_sources.sql

```diff
@@ -66,6 +66,9 @@
             
         {% endfor %}
         
+        {% set update_columns = dbt_external_tables.update_external_table_columns(node) %}
+        {{ update_columns }}
+
```

Comment:
1. Yes, it just updates the schema with operations that can only be done via the API, see:
https://github.com/dbt-labs/dbt-bigquery/blob/6c0afe4cfb69761dada5d16150fe632b8f72bf39/dbt/adapters/bigquery/impl.py#L609
It adds descriptions and policyTags effectively creating the same behaviour as when you create a normal model.

2. So I added the default__update_external_table_columns because I thought that this was how you should implement a macro that is BigQuery specific.

Reply:
perhaps i was looking for something more conventional (for this repo/ dbt macros) like
1. fetching config,
2. if config is not empty, then do the thingy

```suggestion
       {% set update_columns = dbt_external_tables.update_external_table_columns(node) %}
        {%- if  update_columns -%}
            {{ update_columns }}
```